wallet/src/Pos/Wallet/Web/Tracking/Sync.hs

```diff
@@ -278,12 +280,12 @@ syncWalletWithGStateUnsafe encSK wTipHeader gstateH = setLogger $ do
     whenNothing_ wTipHeader $ do
         let encInfo = getEncInfo encSK
             ownGenesisData =
-                selectOwnAccounts encInfo (txOutAddress . toaOut . snd) $
+                selectOwnAddresses encInfo (txOutAddress . toaOut . snd) $
```

Comment:
Comes from [this section](https://github.com/input-output-hk/cardano-sl/commit/fee01810a33d73ebf349a0995efc0dd74ced6f58#diff-b94d2b54606d6df6da3ad7f83a987aceR283) -- the usual patch "[CSM-494] + [CSM-505]".